decayAmplitude/massDependence.h

```diff
@@ -415,6 +426,75 @@ namespace rpwa {
 	typedef boost::shared_ptr<rhoPrimeMassDep> rhoPrimeMassDepPtr;
 
 
+	//////////////////////////////////////////////////////////////////////////////
+	/// Brief GLASS parameterization for [Kpi]_S wave
+	/// Taken from The European Physical Journal C 2014 vol: 74 (11) pp: 3026
+	class KPiSGLASS: public massDependenceImpl<KPiSGLASS> {
+
+	public:
+
+		KPiSGLASS(const double a, const double r, const double M0, const double G0, const double phiF, const double phiR, const double phiRsin, const double F,
+		          const double R, const double MMax, const std::string& tag);
+		~KPiSGLASS()                        { }
+
+		using massDependenceImpl<KPiSGLASS>::Create;
+		static boost::shared_ptr<KPiSGLASS> Create(const libconfig::Setting* settings);
+
+		virtual std::string parentLabelForWaveName(const isobarDecayVertex& v) const;  ///< returns label for parent of decay used in wave name
+
+		virtual std::complex<double> amp(const isobarDecayVertex& v);
+
+		static constexpr const char* cName = "KPiSGLASS";
```

Comment:
general comment: shouldn't it be _cName throughout for consistency?

Reply:
I do not think so. 
It is a static **public** member.